packages/taquito-contracts-library/src/rpc-wrapper.ts

```diff
@@ -46,6 +46,8 @@ import {
   VotingPeriodBlockResult,
   TicketTokenParams,
   AllTicketBalances,
+  PendingOperations,
+  PendingOperationsQuery,
```

Comment:
should this follow the same conventions of `params`?

Reply:
it's a url query that looks like `?key=value` tailing a url. There so the name but will update it to follow the convention of `BakingRightsQueryArguments` and `EndorsingRightsQueryArguments`